src/components/ReactTable/ReactTable.tsx

```diff
@@ -147,22 +147,13 @@ export const ReactTable = <T extends {}>({
     visibleColumns.length,
   ])
 
-  const scrollToTableTop = useCallback(() => {
-    if (tableRef.current) {
-      const scrollY = tableRef.current.offsetTop - 80
-      window.scrollTo(0, scrollY)
-    }
-  }, [])
-
   const handlePrevious = useCallback(() => {
     previousPage()
-    scrollToTableTop()
-  }, [previousPage, scrollToTableTop])
+  }, [previousPage])
 
   const handleNext = useCallback(() => {
     nextPage()
-    scrollToTableTop()
-  }, [nextPage, scrollToTableTop])
+  }, [nextPage])
```

Comment:
`handleNext` and `handlePrevious` can be removed and simply pass in `nextPage` and `previousPage` directly to the consumer